BUILD.gn

```diff
@@ -241,9 +241,19 @@ group("create_dist") {
         "build/mac:unsigned_dmg"
       ]
     } else {
-      deps += [
-        "build/mac:finalize_dmg_pkg"
-      ]
+      deps += [ "build/mac:finalize_dmg_pkg" ]
+
+      if (sparkle_eddsa_private_key != "") {
+        deps += [ "build/mac:generate_eddsa_sig_for_dmg" ]
+
+        if (build_delta_installer) {
```

Comment:
building delta shouldn't depend on private key but signing delta should

Reply:
got it.